downstream/modules/platform/ref-postgresql-requirements.adoc

```diff
@@ -29,15 +28,12 @@ h| Service |Required |Notes
 * All {ControllerName} data is stored in the database.
 Database storage increases with the number of hosts managed, number of jobs run, number of facts stored in the fact cache, and number of tasks in any individual job.
 For example, a playbook run every hour (24 times a day) across 250 hosts, with 20 tasks, will store over 800000 events in the database every week.
-* If not enough space is reserved in the database, old job runs and facts must be cleaned on a regular basis.
-Refer to link:https://docs.ansible.com/ansible-tower/3.8.3/html/administration/management_jobs.html#ag-management-jobs[Management Jobs] in the _Automation Controller Administration Guide_ for more information
-
+* If not enough space is reserved in the database, old job runs and facts must be cleaned on a regular basis. Refer to link:https://docs.ansible.com/ansible-tower/3.8.3/html/administration/management_jobs.html#ag-management-jobs[Management Jobs] in the _Automation Controller Administration Guide_ for more information.
```

Comment:
Very minor; seems like an article is needed. You can ignore this suggestion if you feel it's unnecessary.

```suggestion
* If not enough space is reserved in the database, the old job runs and facts must be cleaned on a regular basis. Refer to link:https://docs.ansible.com/ansible-tower/3.8.3/html/administration/management_jobs.html#ag-management-jobs[Management Jobs] in the _Automation Controller Administration Guide_ for more information.
```